src/domain/document-type.ts

```diff
@@ -2,16 +2,19 @@ export interface IDocumentType {
   id: string;
   title: string;
   description?: string;
+  enabled: boolean;
```

Comment:
Added new prop -- the response from the API will either return doc types with enabled being true or false